base/server/scripts/pki-server-nuxwdog

```diff
@@ -122,3 +122,7 @@ for tag in sorted(iter(tags)):
     key_name = instance_name + '/' + tag
 
     keyring.put_password(key_name=key_name, password=entered_pass)
+
+# 4. Put this script to sleep in background to keep the keyring fd open until main program starts
+# due to systemd bug #1668954
+subprocess.Popen(['/usr/bin/sleep', '10'])
```

Comment:
My previous comment seems to have disappeared. I was asking why not use time.sleep(10)?

Reply:
`time.sleep` stops the code execution temporarily. But, the above code creates a new process with a new PID, which basically dies after 10 seconds. The need for this new process is to keep at least ONE process running for `pkiuser` between `ExecStartPre` and `ExecStart`. 

Related question in systemd-devel: https://lists.freedesktop.org/archives/systemd-devel/2019-January/042044.html

Reply:
Sounds good. Approving..